configure.ac

```diff
@@ -397,14 +397,14 @@ AC_ARG_ENABLE(request-early,
 
 # Enable duplicate rules id
 AC_ARG_ENABLE(unique_id,
-              AS_HELP_STRING([--enable-not_unique_id],
+              AS_HELP_STRING([--enable-rule-id-validation],
                              [Allow duplicate rule ids and no id. default is to disable that]),
```

Comment:
I think you forgot to commit the help message. It seems that it describe the negative of what the parameter is actually doing. The default is to avoid duplicate IDs, not otherwise. If you disable the check, than, you will be able to use duplicate IDs. Right?